test/annexB/built-ins/RegExp/named-groups/non-unicode-malformed.js

```diff
@@ -17,11 +17,9 @@ assert(/\k/.test("k"));
 
 assert(/(?<a>\a)/.test("a"));
 
-assert(compareArray(["k<a>"], "xxxk<a>xxx".match(/\k<a>/)));
-assert(compareArray(["k<a"], "xxxk<a>xxx".match(/\k<a/)));
+assert.compareArray(["k<a>"], "xxxk<a>xxx".match(/\k<a>/));
+assert.compareArray(["k<a"], "xxxk<a>xxx".match(/\k<a/));
 
-// A couple of corner cases around '\k' as named back-references vs. identity
-// escapes.
-assert(/\k<a>(?<=>)a/.test("k<a>a"));
-assert(/\k<a>(?<!a)a/.test("k<a>a"));
 assert(/\k<a>(<a>x)/.test("k<a><a>x"));
+assert(/\k<a>\1/.test("k<a>\x01"));
+assert(/\1(b)\k<a>/.test("bk<a>"));
```

Comment:
while the assertions are fine, it seems it would be slightly better to capture the results if they were positioned in a new test file, for similar reasons some assertions here are being set in a new file.